### PR TITLE
Update English Vokuro page.

### PR DESCRIPTION
**Pull request by dschissler**
This one is better now but the other languages are largely very out of date.  I'm wondering if they should all be overwritten by the English page and then updated as we can get access to a speaker of those languages.

### CODE REVIEW

File: en/reference/tutorial-vokuro.rst [hunk 1]
```diff
@@ -54,8 +78,8 @@ the composer.json looks like:
     {
         "require" : {
             "php" : ">=5.4.0",
-            "ext-phalcon" : ">=2.0.0",
```
Comment (sergeyklay) [inline en/reference/tutorial-vokuro.rst]: `"php" : ">=5.5.0",`

**Comment by dschissler** [comment]
There is certainly a lot of non-English translations that can be recovered but I determined that the translations that are intermixed in with code are largely unrecoverable for someone who does not know that language.  Perhaps we could track the files and try to alert some people.  Some descriptions appear to be easy enough to recover but the Eastern Asian languages are just a wild guess really.

**Comment by dschissler** [comment]
I'm calling it a night tonight.   In some days Icould recovery some of the Latin language based descriptions.

**Comment by SidRoberts** [comment]
Could you restore the translations in pt and ru? They appear to be correct.

**Comment by SidRoberts** [comment]
Merged in 20e983dfd6123bf4eeb560522995182422ffd0a7.

**Comment by dschissler** [comment]
@SidRoberts How can we get the documentation on the website updated?  The current version that is up is very wrong about for Vokuro.

**Comment by dschissler** [comment]
@andresgutierrez @sergeyklay How to update documentation on website?  This may not be perfect but the current docs are broken for Vokuro.

**Comment by sergeyklay** [comment]
@dschissler Not deployed yet

**Pull request closed without merging**